src/main/java/Main.java

```diff
@@ -7,11 +9,11 @@ public static void main(String[] args) {
         // test logic
 
         List list = Arrays.asList("hello","world");
-
+        String str = "";
         for(int i = 0; i<list.size(); i++){
-            System.out.println(list.get(i));
+            str += list.get(i);
         }
-        //
+        System.out.println(str.toString());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHelloWorld&pullRequest=21&issues=AWQNbL-AuoX1TGVkTl-b&open=AWQNbL-AuoX1TGVkTl-b)

[ISSUE_KEY]: AWQNbL-AuoX1TGVkTl-b